### ISSUE TITLE
Syntax for single and multiple access tokens

### ISSUE THREAD

**Issue by jricher** [ietf-wg-gnap/gnap-core-protocol#68]
§3.2 Access Tokens: Editor's Note:

> The AS MUST NOT respond with both the access_token and multiple_access_token fields.

I really don't like the dichotomy between "access_token" and "multiple_access_tokens" and their being mutually exclusive, and I think we should design away from this pattern toward something less error-prone.

**Comment by aaronpk** [comment]
Can’t this follow the pattern of the rest of the spec where a value may be a single object or an array?


```
{
  “access_token”: {
    “value”: “XXXX”
  }
}
```

vs 

```
{
  “access_token”: [
    {
      “id”: “token1”,
      “value”: “XXXX”
    },
    {
      “id”: “token2”,
      “value”: “YYYY”
    }
  ]
}
```

**Comment by jricher** [comment]
This is likely to follow the format of the request, such as that proposed in #40

**Comment by jricher** [comment]
closed by #162
